src/controllers/controlpickermenu.cpp

```diff
@@ -243,6 +243,12 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
     addDeckAndSamplerControl("bpm_up_small", tr("BPM +0.1"), tr("Increase BPM by 0.1"), pBpmMenu);
     addDeckAndSamplerControl("bpm_down_small", tr("BPM -0.1"), tr("Decrease BPM by 0.1"), pBpmMenu);
     addDeckAndSamplerControl("bpm_tap", tr("BPM Tap"), tr("BPM tap button"), pBpmMenu);
+    addDeckAndSamplerControl("beats_set_halve", tr("BPM Halve"), tr("BPM halve"), pBpmMenu);
```

Comment:
```suggestion
    addDeckAndSamplerControl("beats_set_halve", tr("BPM Halve"), tr("Multiply current BPM with 0.5"), pBpmMenu);
```
etc.

Reply:
Taking a second look, I think it would be better to have the actions descried like the occur in the track menu
halve - 2/3 - 3/4 - 4/3 - 3/2 - double
https://github.com/mixxxdj/mixxx/blob/3aa73fd1cb0f39c3993bff2913f72c7d03c58c13/src/widget/wtrackmenu.cpp#L454